.travis.yml

```diff
@@ -4,22 +4,26 @@ language: cpp
 
 addons:
   apt:
+    sources:
+      - sourceline: 'ppa:beineri/opt-qt596-xenial'
     packages:
+      - libgl-dev
```

Comment:
Could you please explain why do we need OpenGL support?

Reply:
> Could you please explain why do we need OpenGL support?

Because Qt5.9 in Ubuntu Xenial seems to require GL as we can see [here](https://travis-ci.org/ihhub/penguinV/jobs/539579691#L540).

Reply:
Ah, okay. Thanks for explanation.